kaolin/ops/conversions/pointcloud.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2019-2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+
+__all__ = ['pointclouds_to_voxelgrids']
+
+def pointclouds_to_voxelgrids(pointclouds, resolution, return_sparse=False):
```

Comment:
shouldn't we adopt the same interface than `trianglemesh_to_voxelgrids` ? https://github.com/NVIDIAGameWorks/kaolin/blob/master/kaolin/ops/conversions/trianglemesh.py#L20

Reply:
Do you mean we should also add origin and scale as arguments?

Reply:
Yes!

Reply:
Added, as well as the tests.